v3/src/models/data/formula-utils.test.ts

```diff
@@ -22,7 +24,7 @@ const displayNameMapExample: DisplayNameMap = {
     "Roller Coaster": {
       "id": "DATA_ROLLER_COASTER",
       "attribute": {
-        // \, ', and " are added to test characters escaping
+        // \, ", and " are added to test characters escaping
```

Comment:
Seems like this makes the comment redundant -- was this change intended?

Reply:
Thanks, it wasn't, probably some search-and-replace mistake. Fixed in the last commit.